### PR TITLE
chore: enable manual start of btc integration test workflow 

### PR DESCRIPTION
**Pull request by pavitthrap**
### Description
Updated the `stacks-bitcoin-integration-tests` workflow file to allow running the workflow through the GitHub UI. This can be helpful if a workflow was manually stopped for any reason.

### CODE REVIEW

File: .github/workflows/bitcoin-tests.yml [hunk 1]
```diff
@@ -2,8 +2,10 @@ name: stacks-bitcoin-integration-tests
 
 # Only run when:
 #   - PRs are opened
+#   - the workflow is started from the UI
 on:
   pull_request:
+  workflow_dispatch:
```
Comment (igorsyl) [inline .github/workflows/bitcoin-tests.yml:8]: Nice!
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

**Review by igorsyl — APPROVED** [review]
Should we add a note to contributors readme on how to trigger the btc int tests?

File: .idea/stacks-blockchain.iml [hunk 2]
```diff
@@ -10,9 +10,10 @@
       <sourceFolder url="file://$MODULE_DIR$/stx-genesis/src" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/testnet/puppet-chain/src" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/testnet/stacks-node/src" isTestSource="false" />
+      <sourceFolder url="file://$MODULE_DIR$/contrib/tools/relay-server/src" isTestSource="false" />
```
Comment (gregorycoppola) [inline .idea/stacks-blockchain.iml:13]: i checked in this `.idea` directory :/... @donpdonp was talking about deleting this

can this file at least be dropped for now?

**Review by gregorycoppola — COMMENTED** [review]
one comment

File: .idea/stacks-blockchain.iml (hunk 2 above)
Comment (gregorycoppola) [reply .idea/stacks-blockchain.iml:13]: update: https://github.com/stacks-network/stacks-blockchain/pull/3523

Comment (gregorycoppola) [reply .idea/stacks-blockchain.iml:13]: Oh.. well #3523 is open against `develop` right now and this PR is against `next`

**Review by gregorycoppola — APPROVED** [review]

**Pull request merged**